lib/tasks/data_key_validation.rake

```diff
@@ -0,0 +1,52 @@
+
```

Comment:
i see that we already have a `rake` task named `data-key.rake`. how about we add a task `:validate` into that file instead of creating a new file?